R/average_wait.R

```diff
@@ -1,16 +1,20 @@
 #' @title Title
```

Comment:
Would it be possible to add a title along the lines of:  Calculate the average waiting time

Reply:
Oh sorry - missed that!

Reply:
It's done now so just needs approving to merge - thanks